tools/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $0 VERSION"
+  echo ""
+  echo "Expected to be run from the root of the repo"
+  exit 1
+fi
+
+version="$1"
+mkdir td-grpc-bootstrap-${version}/
```

Comment:
Use quotes please (and below), since you don't validate `version` and it could have special characters:
```
mkdir "td-grpc-boostrap-${version}/"
```

Reply:
Done. This started as just these 4 lines since they wouldn't fit all that nicely in the yaml. And then it grew from there as I wrote echos and such instead of comments.